staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -101,6 +111,26 @@ func logError(err error) {
 	glog.ErrorDepth(2, err)
 }
 
+type rudimentaryErrorBackoff struct {
+	// TODO(lavalamp): use the clock for testability. Need to move that
+	// package for that to be accessible here.
+	lastErrorTime time.Time
+	minPeriod     time.Duration
+	lock          sync.Mutex
+}
+
+// OnError will block if it is called more often than the embedded period time.
+// This will prevent overly tight hot error loops.
+func (r *rudimentaryErrorBackoff) OnError(error) {
```

Comment:
Now we aren't even going to log the error? Is this responsible for more error logs then just the one we were seeing in logspam?